src/bin/solang.rs

```diff
@@ -589,6 +599,26 @@ fn contract_results(
         return;
     }
 
+    if ns.top_file_no() != resolved_contract.loc.file_no() {
+        // contracts which were imported should not be considered. For example, if we have a file
```

Comment:
```suggestion
        // contracts that were imported should not be considered. For example, if we have a file
```

Reply:
If you use `which`, you have a non-restrictive clause. `contracts, which were imported, should not be considered` means that all contracts were imported and all of them will not be considered.
When you use `that`, you have a restrictive clause. `contracts that were imported should not be considered` means that you are only not considering imported contract .

https://academicguides.waldenu.edu/writingcenter/grammar/clauses